openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.tsx

```diff
@@ -181,7 +181,7 @@ export const CustomPropertyTable: FC<CustomPropertyProps> = ({
   }, [entityType]);
 
   if (entityTypeDetailLoading) {
-    return <Loader />;
+    return <Skeleton active />;
```

Comment:
Do we need this now?

Reply:
Yes here we are rendering the table dynamically so we do not have any fixed headers, that's why we need skeleton loader directly.